operator/v1/types_machineconfiguration.go

```diff
@@ -56,3 +65,98 @@ type MachineConfigurationList struct {
 	// Items contains the items
 	Items []MachineConfiguration `json:"items"`
 }
+
+type ManagedBootImages struct {
+	// machineManagers is an array of machineManager objects.
+	// The MCO will watch for changes to this list and register/de-register machine management resources from boot image updates.
+	// An entry in this list consists of the resource type, the API group that the resource belongs to and a selection filter
+	// on the resources.
+	//
+	// Warning: Only one entry is permitted per unique pair of resource/API group. The label selector provided within MachineManager
+	// can be used for further customization if required.
+	//
+	// +optional
+	// +listType=map
+	// +listMapKey=resource
+	// +listMapKey=apiGroup
+	MachineManagers []MachineManager `json:"machineManagers"`
+}
+
+// MachineManager contains identifying information of a machine management resource(eg. a machineset) that will be
+// registered for boot image updates. This is likely to evolve as support for more machine management resources are added.
+type MachineManager struct {
+	// resource is the machine management resource's type.
+	//
+	// The following values are accepted:
+	// - MachineSets: The machine manager will only register resources of the type MachineSet, which may belong to MachineAPI or ClusterAPI.
```

Comment:
Casing matters, this should be lowercased
```suggestion
	// - machinesets: The machine manager will only register resources of the type MachineSet, which may belong to MachineAPI or ClusterAPI.
```

Reply:
Ahh, I accidentally put in the type names instead of the string. Will fix